tripy/tripy/frontend/module/layernorm.py

```diff
@@ -47,7 +54,10 @@ class LayerNorm(Module):
     eps: float
     """A value added to the denominator to prevent division by zero."""
 
-    def __init__(self, normalized_shape: int, dtype: datatype.dtype = datatype.float32, eps: float = 1e-5) -> None:
+    correction: int
+    """Difference between the sample size and the degrees of freedom."""
```

Comment:
Let's specify what it defaults to (correction=0 which is the biased estimator for variance).

Reply:
Removed correction.